docs/admin/access.rst

```diff
@@ -9,62 +9,51 @@ Access control
     specifically built for Weblate. If you are using anything older, please consult
     the documentation for the specific version you are using.
 
-Weblate comes with a fine-grained privilege system to assign user permissions
-for the whole instance, or in a limited scope.
+After installing Weblate, you can grant permissions to various things on the
+whole installation using :ref:`default-groups`. :ref:`acl` lets you grant
+permissions to specific translation projects. If you want to fine-tune the
+permissions you give based on groups you make yourself, you can do so
+using :ref:`custom-acl`.
 
-The permission system is based on groups and roles, where roles define a set of
-permissions, and groups assign them to users and translations, see
-:ref:`auth-model` for more details.
-
-After installation a default set of groups are created, and you can use those
-to assign users roles for the whole instance (see :ref:`default-groups`). Additionally when
-:ref:`acl` is turned on, you can assign users to specific translation projects.
-More fine-grained configuration can be achieved using :ref:`custom-acl`.
-
-Common setups
--------------
-
-Locking down Weblate
-++++++++++++++++++++
-
-To completely lock down your Weblate, you can use :setting:`REQUIRE_LOGIN` to
-force users to sign in and :setting:`REGISTRATION_OPEN` to prevent new
-registrations.
+In the permission system, roles define a set of permissions, and you can
+put users in groups that have one or more of these roles, described in
+:ref:`auth-model`.
 
 Site wide permissions
 +++++++++++++++++++++
 
-To manage permissions for a whole instance, just add users to `Users` (this is done
-by default using the :ref:`autogroup`), `Reviewers` and `Managers` groups. Keep
-all projects configured as `Public` (see :ref:`acl`).
+By default, new users get access to comment, provide suggestions, and make translations
+everywhere by being put in the `Users` group automatically by :ref:`autogroup`.
+You can put the users you deem fit in the `Reviewers` to let them review everything,
+and the `Managers` groups to do all of the above everywhere.
```

Comment:
The whole section looks irrelevant for the title: doesn't tell anything on how to manage the site-wide permissions.

Also it is misleading:
* `Reviewers` don't review everything, but just components configured to be reviewed.
* `Managers` has much more rights than listed above.

Reply:
@Fat-Zer what I was going for might work better with: "You can put the users you deem fit in groups such as `Users`". Is anything missing then?